### ISSUE TITLE
As a SiteNow developer, I can deploy a theme that uses UIDS with paragraphs (part two)

### ISSUE THREAD

**Issue by joewhitsitt** [uiowa/uiowa#1660]
Following up from https://github.com/uiowa/uiowa/issues/1324
- [x] #1654 Account for background color options with paragraph bundles
- [x] Maybe covered by 1654, but dark gray bg with yellow links fails contrast ratio.
- ~~Support for page title `sub_title` (e.g. Person credentials, etc.)~~ _With the person credentials accounted for, there currently isn't another use case for sub_title right now_
- [x] https://github.com/uiowa/uiowa/issues/1617 Sidebar menus
- [x] #1662 wysiwyg stylesheets
- [x] https://github.com/uiowa/uids/issues/199, https://github.com/uiowa/uids/issues/194 IE11
- [x] https://github.com/uiowa/uiowa/issues/1619 Spacing above footer
- [x] https://github.com/uiowa/uiowa/issues/1633 Header adjustments
- [x] https://github.com/uiowa/uiowa/issues/1678 Preview header/actions
- [x] https://github.com/uiowa/uids/issues/123 UIDS Carousel
- [x] https://github.com/uiowa/uiowa/issues/1465 Events 
- [x] Featured Content
- [x] https://github.com/uiowa/uiowa/pull/1468 People
- [x] https://github.com/uiowa/uiowa/issues/1611 Migrate to new footer menus _Complete, waiting for deployment_
- [x] https://github.com/uiowa/uiowa/issues/1749 ighn.international.uiowa.edu
- [x] hr.uiowa.edu has been accounted for and will not break with this rollout. communications about adoption continue.
- [x] https://github.com/uiowa/uiowa/issues/1756 Reorganize sitenow directory within uids_base 
- [x] https://github.com/uiowa/uiowa/issues/1137 visual indication of unpublished content
- [x] #1811 Footer logo appears smaller in SiteNow sites
- [x] #1808 styling for nonlink <a> tags used for anchors
- [x] #1810 styling (primarily line-height for spacing) of text in `<div>` tags
- [x] https://github.com/uiowa/uiowa/issues/1812 Loss of default top padding on some paragraphs
- [x] https://github.com/uiowa/uiowa/issues/1801 Toggle menu issue.
- [x] https://github.com/uiowa/uiowa/issues/1815 Article display issues 
- [x] https://github.com/uiowa/uiowa/issues/1844 Article mobile display regression
- [x] #1817 Migrate uiowa_bar quick links
- [x] https://github.com/uiowa/uiowa/issues/1822 Restrict top links to two on non-homepage sites
- [x] #1820 'Additional Information' content from basic site settings
- ~~#1824 Site slogan, lockup unused/remains in basic site settings~~ _Based on current decisions, this can happen at a later date_
- [ ] #1775 a11y fixes
- [x] Communicate the upcoming change (alert, article, listserv, sprint demo) and provide docs about manual clean-up: https://github.com/uiowa/uiowa/issues/1401

There are possibly several other issues that need to be addressed before uids_base adoption happens for sites. Pick out random sites to pull down, and turn on uids_base locally. Create and relate issues to this one to be addressed.

**Comment by joewhitsitt** [comment]
Unless someone has other ideas, I think the paragraph bundles events, featured content, people should all follow the same design pattern currently being implemented for paragraph articles in uids_base as they are all currently similar in uiowa_bootstrap. This is at least a short-term solution to speed up the adoption of uids_base until they can be revisited if needed.

**Comment by bspeare** [comment]
@joewhitsitt That sounds like a good plan.
